### ISSUE TITLE
ElvUI plugin deleted :(

### ISSUE THREAD

**Issue by Macumbatbc** [fuba82/SonsOfTheForestTools#1]
Hello,
Sorry to put this here, i saw you deleted all the wow addons you had.
Do you still have them? Many poeple still use the ElvUI plugins you had...

Could you send me what your addons please ? I could reupload them.

Thanks in advance.

**Comment by fubaWoW** [comment]
hi there,

I've moved all my Wow Addons to https://github.com/fubaWoW?tab=repositories

What ElvUI Plugins you're actually looking for?

**Comment by Macumbatbc** [comment]
Hey,

Thanks ! I forgot to say it was addons for TBC 2.4.3
I had a list of the addons you had :
ElvUI_Ammo-TBC
ElvUI_BagFilter-TBC
ElvUI_BagsLeft-TBC
ElvUI_Professions-TBC
ElvUI_RunSpeed-TBC
fuba_LinkGrabber-TBC
fuba_DepositBank-TBC
GatherSage-Enhanced-TBC
And Molinari (but it wasn't working afaik)




________________________________

**Comment by fubaWoW** [comment]
ah ok, that is another thing!

That addons was Designed to work (some of that was still in Progress or not fully Fixed) for the OLD Burning Crusade, especially for Private Servers!
As of the Release of the Burning Crusade Classic i've removed all of them, because it was no longer needed to play on Private Servers and the "new" Addon-API is different and could cause Errors.

BUT you can download my Backup from here if you want:
[GitHub_TBC_Addons.zip](https://github.com/fuba82/SonsOfTheForestTools/files/10938847/GitHub_TBC_Addons.zip)

or here, if the link above would not work (7Zip-File):
https://we.tl/t-GjlFzjdyXv

I think this Package should contain all of them.

**Comment by Macumbatbc** [comment]
Thank you so much !
I'll check that and i understand, i played on classic too, but now who want to play on vanilla classic can still play the game. That's not the case for TBC lovers unfort :(
That's why no choice to play on pserv atm...

**Comment by fubaWoW** [comment]
Yeah i also don't know why they just upgraded ALL Servers to WotlK Classic...
But that is Blizzard ^^

**Comment by Macumbatbc** [comment]
Yeah it seems that's why there is less poeple on classic, because "that's blizzard"
And if you remove the L on blizzard, it become "bizzard" which mean strange in french. That's what they are. :)
